docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,384 @@
+       
+3. Install Teleport on each instance.
+
+   ```bash
+   # Install Teleport
+   curl https://deb.releases.teleport.dev/teleport-pubkey.asc | sudo apt-key add -
+   sudo add-apt-repository 'deb https://deb.releases.teleport.dev/ stable main'
+   sudo apt-get update
+   sudo apt-get install teleport
+   ```
+
+4. Configure Teleport to use ACME and to automatically acquire a TLS certificate:
+
+   <Admonition type="tip" title="Tip">
+    Teleport will now automatically acquire an ACME-compliant TLS certificate by default. While you can use a third-party ACME client (like Let's Encrypt certbot), the configuration commands below will do the same.
+   </Admonition>
+
+   ```bash
+   # Configure Teleport
+   sudo teleport configure \
+     --acme --acme-email=your_email@example.com \
+     --cluster-name=auth.example.com \
+     -o file
+   ```
+
+5. Launch Teleport on `auth.example.com`: `sudo teleport start`. 
+
+## Step 1b. Add a Node to the cluster
+
+<video autoplay loop muted playsinline>
+  <source src="https://goteleport.com/teleport/videos/server-access/addnode.mp4" type="video/mp4"/>
+  <source src="https://goteleport.com/teleport/videos/server-access/addnode.webm" type="video/webm"/>
+Your browser does not support the video tag.
+</video>
+
+1. Now, create a third terminal and connect to `auth.example.com`:
+
+   i. Create a user to access the `auth.example.com` Web UI: 
+
+   - `sudo tctl users add auth-admin --roles=editor,access,admin --logins=root,ubuntu,ec2-user`  
+
+   ii. This will generate an initial login link where you can set a password and set up Two-Factor Authentication for `auth-admin`.
+
+   iii. Create a *join token* to link `ssh.example.com` with the `auth.example.com` cluster:
+
+   - `sudo tctl tokens add --type=node`  
+   - "Node" mode specifies that the Teleport Node that will join will act as an SSH server.  
+
+   iv. You should see:
+
+   ```bash
+   # Join cluster
+   sudo teleport start \
+     --roles=node \
+     --token=d62629ea83e181644e0778af22ec3a8e \
+     --ca-pin=sha256:c75f8be5b21a5cc1b6f91a71200f30a9a083dfea4fc47ec1c55143ccce8882e4 \
+     --auth-server=172.31.17.124:3025
+   ```
+
+   <Admonition type="warning" title="Warning">
+     Remember to replace the IP address supplied by default above with the `auth.example.com` proxy address: `auth.example.com:443`.
+   </Admonition>
+
+2. On your second terminal, `ssh.example.com`:
+
+   i. Run the command above with the correct `auth.example.com` public proxy address on the machine you wish to connect.
+
+   ii. You should now be able to view both Nodes at: `https://auth.example.com/web/cluster/auth.example.com/nodes` in the Web Interface after logging in with your the user `auth-admin`.
+
+   ![Both Nodes in](../../img/server-access/teleport_ui.png)
+
+## Step 2a. Use tsh to connect to the cluster
```

Comment:
I would lead with the use-case that users can relate to, and then explain the tool below:

* Step 2/n SSH into the server